cmd/util/cmd/verify_execution_result/cmd.go

```diff
@@ -93,9 +93,9 @@ func run(*cobra.Command, []string) {
 }
 
 func parseFromTo(fromTo string) (from, to uint64, err error) {
-	parts := strings.Split(fromTo, "-")
+	parts := strings.Split(fromTo, "_")
```

Comment:
The `from_to` help message and delimiter should match.  Currently, the help message uses `-` (instead of `_`) in examples.

https://github.com/onflow/flow-go/blob/f7188ad854105e4c938ea31b48c9204fc81e9b3b/cmd/util/cmd/verify_execution_result/cmd.go#L49-L50